test/integration/features/cmd-docker-env.feature

```diff
@@ -26,9 +26,8 @@ one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_SPECIFIED_SHELL parame
 
   Scenario: Unsetting Docker client using --unset flag of docker-env command
      When executing "minishift docker-env --unset" in host shell succeeds
-      And evaluating stdout of the previous command in host shell
+     Then evaluating stdout of the previous command in host shell
```

Comment:
This really should be `and`. I run unset the docker-env and I execute the command it prints to the console and I execute `docker info`. Then only I can expect something and that is that the output of `docker info` does not contain `Name: minishift`.

Reply:
```
When
  And
 Then
  And
```
@agajdosi Is this the ordering you are looking for ? I feel this is better that the previous order

Reply:
@amitkrout , yes it is 
```
When
  And
  And (if similar test case)
Then
```

Reply:
In this case I think it should be as @budhram mentioned:
```
When
  And
  And (if similar test case)
Then
```
`And executing "docker info" in host shell` is not something we check, it is just part of WHEN (prolonged with AND), it is preparation, after witch THEN comes in which we actually really care what is the result, this is the actual check, why we do all of this.